package.json

```diff
@@ -0,0 +1,130 @@
+    "styled-components": "^3.x.x"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-do-expressions": "^7.0.0",
+    "@babel/plugin-proposal-export-default-from": "^7.0.0",
+    "@babel/plugin-proposal-export-namespace-from": "^7.0.0",
+    "@babel/plugin-proposal-json-strings": "^7.0.0",
+    "@babel/plugin-proposal-logical-assignment-operators": "^7.0.0",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.0.0",
+    "@babel/plugin-proposal-numeric-separator": "^7.0.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
+    "@babel/plugin-proposal-pipeline-operator": "^7.0.0",
+    "@babel/plugin-proposal-throw-expressions": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-syntax-import-meta": "^7.0.0",
+    "@babel/plugin-transform-react-jsx": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/runtime": "^7.0.0",
+    "@integreat-app/eslint-config-babel": "^0.3.1",
+    "@integreat-app/eslint-config-flow": "^0.3.2",
+    "@integreat-app/eslint-config-jest": "^0.2.1",
+    "@integreat-app/eslint-config-node": "^0.3.2",
+    "@integreat-app/eslint-config-react": "^0.3.3",
+    "assets-webpack-plugin": "^3.9.6",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-eslint": "^9.0.0",
+    "babel-jest": "^23.6.0",
+    "babel-loader": "^8.0.0",
+    "babel-plugin-styled-components": "^1.5.1",
+    "browser-sync": "^2.24.7",
+    "connect-history-api-fallback": "^1.5.0",
+    "ejs": "^2.6.1",
+    "enzyme": "^3.5.0",
+    "enzyme-adapter-react-16": "^1.3.0",
+    "enzyme-to-json": "^3.3.4",
+    "es-check": "^2.1.0",
+    "escape-string-regexp": "^1.0.5",
+    "eslint": "^5.8.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-config-standard-react": "^7.0.2",
+    "eslint-plugin-flowtype": "^3.1.4",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jest": "^21.26.2",
+    "eslint-plugin-node": "^8.0.0",
+    "eslint-plugin-prefer-object-spread": "^1.2.1",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-react": "^7.11.1",
+    "eslint-plugin-standard": "^4.0.0",
+    "file-loader": "^2.0.0",
+    "flow-bin": "^0.79.1",
+    "flow-copy-source": "^2.0.2",
+    "flow-typed": "^2.5.1",
+    "git-repo-version": "^1.0.2",
+    "jest": "^23.6.0",
+    "jest-junit": "^5.1.0",
+    "json-loader": "^0.5.7",
+    "prettier-eslint": "^8.8.2",
+    "raf": "^3.4.0",
+    "react": "^16.6.0",
+    "react-dom": "^16.6.0",
+    "react-hot-loader": "^4.3.5",
+    "react-scroll": "^1.7.10",
+    "redux-mock-store": "^1.5.3",
+    "regenerator-runtime": "^0.12.1",
+    "rimraf": "^2.6.2",
+    "s3": "^4.4.0",
+    "style-loader": "^0.22.1",
+    "styled-components": "^3.4.5",
+    "stylefmt": "^6.0.2",
+    "stylelint": "8.4.0",
```

Comment:
I think stylelint is not needed anymore.

Reply:
it is. it lints the css